src/styles/relevant.scss

```diff
@@ -0,0 +1,40 @@
+.relevant {
+  .card-title {
+    font-size: 24px;
+    font-weight: bold;
+    margin-bottom: 4px;
```

Comment:
Would you mind adding a tiny bit more spacing below the title and above the text (to give the date and time a little more space)?

Reply:
![Screen Shot 2021-01-28 at 11 22 30 PM](https://user-images.githubusercontent.com/49101279/106231414-b3e96f80-61bf-11eb-8e81-3214db93df66.png)